samples/WOCCatalog/WOCCatalog/AccelerateViewController.m

```diff
@@ -31,19 +31,26 @@
 #import "AccelerateViewController.h"
 #import "SingleImageViewController.h"
 
+static const double meanDivisor = 100;
+
 @implementation AccelerateViewController
 
 int accelerateImageNumber = 1;
-int valueRed = 100, valueGreen = 100, valueBlue = 100;
+int valueRed = 100;
+int valueGreen = 100;
+int valueBlue = 100;
 int convolveSize = 1;
-NSArray *indexPathArray;
-UIImage *img;
-UIImageView *imv;
+NSArray* indexPathArray;
+UIImage* img;
+UIImageView* imv;
 UISlider* redSlider;
 UISlider* greenSlider;
 UISlider* blueSlider;
 UISlider* convolveSlider;
-UILabel *rLabel, *gLabel, *bLabel, *boxLabel;
+UILabel* rLabel;
+UILabel* gLabel; 
+UILabel* bLabel;
+UILabel* boxLabel;
```

Comment:
I'm somewhat worried that none of these are instance variables -- they're globals